sass/variables-site/_fonts.scss

```diff
@@ -1,9 +1,9 @@
 // Font and typographic variables
 
-$font__body: "Hoefler Text", "Baskerville Old Face", Garamond, "Times New Roman", serif;
-$font__heading: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
-$font__code: Menlo, monaco, Consolas, Lucida Console, monospace;
-$font__pre: "Courier 10 Pitch", Courier, monospace;
+$font__body: "Hoefler Text", "Baskerville Old Face", garamond, "Times New Roman", serif;
+$font__heading: -apple-system, blinkmacsystemfont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
```

Comment:
@ntwb In Chrome, this change is replacing the system font with “Roboto” in the headings font for some reason.

**Roboto:** `$font__heading: -apple-system, blinkmacsystemfont, "Segoe UI", "Roboto"...` 🚫 
![image](https://user-images.githubusercontent.com/709581/48731449-08215e00-ec0b-11e8-92d7-f4919bf0976f.png)

**SF UI:** `$font__heading: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto"...` ✅ 
![image](https://user-images.githubusercontent.com/709581/48731503-2e46fe00-ec0b-11e8-99d2-8a2bbeacc061.png)

Since all other browsers work fine with the camel case name, I think it should be okay to keep the camel case there  even if its just for Chrome ([some background](https://github.com/necolas/normalize.css/issues/665)). 

Besides that, everything else looks good! Could you also run the compile script before your next commit so the `.css` files get tracked as part of the change here? Thanks!

Reply:
```suggestion
$font__heading: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
```

Interesting, it only changed the case of the font name, it shouldn't have done that 😢

Anyways, I've restored the case of the two changes I see above